app/assets/stylesheets/administrate/components/_flashes.scss

```diff
@@ -1,19 +1,19 @@
 @each $flash-type, $color in $flashes {
   .flash-#{$flash-type} {
     background-color: $color;
-    color: mix($black, $color, 60%);
+    color: color-mix(var(--black) 60%, $color);
```

Comment:
These are technically the only css vars left. CSS (as far as I know) does not have any type of map equivalent. Even so, these get replaced with normal CSS vars in the end.